### DISCUSSION TITLE
Selecting a continue or break statement should highlight the encompassing loop

### DISCUSSION THREAD

**Discussion by Smumbo** [microsoft/pylance-release#3719] [Enhancement]
Essentially what the title says. I think it could be helpful to see at a glance which loop any given `break` or `continue` statement corresponds to when it is selected with the cursor.

Rough mockup of how this could look:
![image](https://user-images.githubusercontent.com/38147112/205165743-ac52dbcc-192c-4649-b8c8-2507d153537c.png)

This would also add the reference highlight to the scrollbar/minimap, much like selecting a variable or function.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
